### PR TITLE
Added new actions

### PR DESCRIPTION
**Pull request by cs-utkarsh-surana**
- Added following new actions and its corresponding playbooks:
    - Get Maps List
    - Get Threat Map
    - Get Threat Map By Org ID
    - Get Malware Threat Map
    - Get Malware Threat Map By Org ID
    - Get Threat Actors List
    - Get Threat Actors Categories
    - Get Malware Categories
    - Get Third Party Risk Alert By Alert ID
    - Get Bulk Third Party Risk Alerts
    - Get Identity Novel Exposures Alert By Alert ID
    - Get Bulk Identity Novel Exposures Alerts
    - Create User List
    - Get User Lists
    - Get User List By List ID
    - Get User List Status By List ID
    - Retrieves entities on the list by list ID
    - Add Entity To User List
    - Remove Entity From User List
    - Add IOC To Recorded Future Intelligence Cloud

### CODE REVIEW

File: recorded-future/info.json [hunk 1]
```diff
@@ -3264,6 +3264,1362 @@
...
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      },
+      "parameters": [
+        {
+          "title": "playbook_alert_ids",
```
Comment (pathansam) [inline recorded-future/info.json]: Please provide appropriate title

File: recorded-future/info.json [hunk 2]
```diff
@@ -3264,6 +3264,1362 @@
...
```
Comment (pathansam) [inline recorded-future/info.json]: Please provide appropriate title

File: recorded-future/info.json [hunk 3]
```diff
@@ -3264,6 +3264,1362 @@
...
+        },
+        {
+          "title": "Context",
+          "description": "Specify the context with which to add the entity.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "context",
+          "value": "{\n    \"additionalProp1\": {}\n  }"
```
Comment (pathansam) [inline recorded-future/info.json:4484]: Please remove default value, add in sample playbook only

File: recorded-future/info.json [hunk 4]
```diff
@@ -3264,6 +3264,1362 @@
...
+          "description": "Specify the organization IDs with which to add IOC To Recorded Future Intelligence Cloud. You can specify multiple organization IDs as csv.",
+          "tooltip": "Specify the organization IDs with which to add IOC To Recorded Future Intelligence Cloud. You can specify multiple organization IDs as csv.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "organization_ids"
+        },
+        {
+          "title": "Timestamp",
```
Comment (pathansam) [inline recorded-future/info.json:4563]: Input type is datetime , Needs to change title,

File: recorded-future/operations.py [hunk 5]
```diff
@@ -35,15 +42,18 @@ def make_rest_call(endpoint, config):
...
-    except requests.exceptions.ReadTimeout:
-        raise ConnectorError(
-            'The server did not send any data in the allotted amount of time')
-    except requests.exceptions.ConnectionError:
-        raise ConnectorError('Invalid endpoint or credentials')
+    except req_exceptions.SSLError:
+        logger.error('An SSL error occurred')
+        raise ConnectorError('An SSL error occurred')
+    except req_exceptions.ConnectionError:
+        logger.error('A connection error occurred')
```
Comment (pathansam) [inline recorded-future/operations.py:49]: Log errors to know actual issue occurred

File: recorded-future/operations.py [hunk 6]
```diff
@@ -246,6 +256,20 @@ def search_alert_rules(config, params):
         raise ConnectorError(str(err))
 
 
+def build_payload(params):
+    data = {}
+    for k, v in params.items():
+        if v:
```
Comment (pathansam) [inline recorded-future/operations.py:262]: Please modify code to handle boolean (False) and integer 0

File: recorded-future/info.json [hunk 7]
```diff
@@ -3264,6 +3264,1362 @@
...
+          "description": "Specify the list ID from which to remove the entity.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "listId"
+        },
+        {
+          "title": "Entity ID",
+          "description": "Specify the entity ID to remove.",
```
Comment (pathansam) [inline recorded-future/info.json:4510]: Please verify is this parameter optional or required

File: recorded-future/operations.py [hunk 8]
```diff
@@ -269,18 +423,23 @@ def test_connection(config):
...
-    except requests.exceptions.ReadTimeout:
-        raise ConnectorError('The server did not send any data in the allotted amount of time')
-    except requests.exceptions.ConnectionError:
-        raise ConnectorError('Invalid endpoint or credentials')
+    except req_exceptions.SSLError:
+        logger.error('An SSL error occurred')
+        raise ConnectorError('An SSL error occurred')
+    except req_exceptions.ConnectionError:
+        logger.error('A connection error occurred')
+        raise ConnectorError('A connection error occurred')
```
Comment (pathansam) [inline recorded-future/operations.py:431]: Log actual error , to know actual error occurred

**Review by pathansam — CHANGES_REQUESTED** [review]

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**